packages/elvis/CHANGELOG.json

```diff
@@ -2,6 +2,24 @@
   "$schema": "../components/changelogSchema.json",
   "name": "elvis",
   "content": [
+    {
+      "version": "19.1.0",
+      "date": "June 20, 2024",
+      "changelog": [
+        {
+          "type": "new_feature",
+          "changes": [
+            "Added new icons <code>ask_help</code>, <code>ask_information</code>, <code>hierarchy</code>, <code>hierarchy-color</code>, <code>more_menu-filled</code>, <code>user_testing</code> and <code>web</code>."
+          ],
+          "pages": [{ "displayName": "Icons", "url": "https://design.elvia.io/tools/icon" }]
+        },
+        {
+          "type": "patch",
+          "changes": ["Updated icon <code>check_bold</code>."],
+          "pages": [{ "displayName": "Icons", "url": "https://design.elvia.io/tools/icon" }]
+        }
+      ]
+    },
```

Comment:
Remember to include the changes from https://github.com/3lvia/designsystem/pull/2954 here before merge